src/main/java/opendcs/dao/DaoBase.java

```diff
@@ -208,7 +208,7 @@ public ResultSet doQuery2(String q) throws DbIoException
 		}
 		try
 		{
-			if (queryStmt2 == null)
+			if (queryStmt2 == null|| statementInvalid(queryStmt1))
```

Comment:
Sounds like you mean to check queryStmt2 here?

Reply:
Wow. good catch.